generators/info/index.js

```diff
@@ -52,25 +52,33 @@ module.exports = ReportGenerator.extend({
             let result = shelljs.cat('.yo-rc.json');
             result = result.replace(/"rememberMeKey": ".*"/g, '"rememberMeKey": "replaced-by-jhipster-info"');
             result = result.replace(/"jwtSecretKey": ".*"/g, '"jwtSecretKey": "replaced-by-jhipster-info"');
-            this.log(`${'\n##### **JHipster configuration, a `.yo-rc.json` file generated in the root folder**\n' +
-                '\n```yaml\n'}${result}\n\`\`\`\n`);
+            console.log(`${'\n##### **JHipster configuration, a `.yo-rc.json` file generated in the root folder**\n' +
+                '\n<details>\n<summary>.yo-rc.json file</summary>\n<pre>\n'}${result}\n</pre>\n</details>`);
             done();
         },
 
         displayEntities() {
             const done = this.async();
-            this.log('\n##### **JDL for the Entity configuration(s) `entityName.json` files generated in the `.jhipster` directory**\n');
-            const jdl = this.generateJDLFromEntities();
-            this.log(`\n\`\`\`yaml\n${jdl.toString()}\n\`\`\`\n`);
+            console.log('\n##### **JDL for the Entity configuration(s) `entityName.json` files generated in the `.jhipster` directory**\n');
+            try {
+                this.getExistingEntities().forEach((entity) => {
```

Comment:
OK I found the link builder in JDL studio so it could be done as below but I wonder if it will work with such long urls. Have you made some tests about the maximum length?

[Sample link with an 8000 bytes long URL](https://jhipster.github.io/jdl-studio/#!#%2Fview%2Fentity%20RankingList%20(ranking_list)%20%7B%0A%20%20period%20Integer,%0A%20%20system%20String,%0A%20%20comments%20String%0A%7D%0Aentity%20TournamentType%20(tournament_types)%20%7B%0A%20%20name%20String,%0A%20%20regional%20Boolean,%0A%20%20universal%20Boolean%0A%7D%0Aentity%20Tournament%20(tournaments)%20%7B%0A%20%20name%20String,%0A%20%20classificationCode%20String,%0A%20%20beginDate%20LocalDate%20required,%0A%20%20endDate%20LocalDate%20required,%0A%20%20country%20String%20required%20minlength(3)%20maxlength(3),%0A%20%20city%20String,%0A%20%20rankingFactor%20Integer,%0A%20%20numberOfNations%20Integer,%0A%20%20numberOfPlayers%20Integer,%0A%20%20ratingWeight%20Float,%0A%20%20logo%20String,%0A%20%20details%20String,%0A%20%20cancelled%20Boolean,%0A%20%20edition%20Integer%20min(1),%0A%20%20detailsFormat%20Integer,%0A%20%20databaseName%20String,%0A%20%20comments%20String%0A%7D%0Aentity%20Continent%20(continents)%20%7B%0A%20%20name_en%20String,%0A%20%20name_fr%20String,%0A%20%20name_es%20String,%0A%20%20name_pt%20String,%0A%20%20name_ru%20String,%0A%20%20created_at%20Instant,%0A%20%20updated_at%20Instant%0A%7D%0Aentity%20Country%20(countries)%20%7B%0A%20%20code%20String%20required%20minlength(3)%20maxlength(3),%0A%20%20isoCode%20String%20minlength(2)%20maxlength(2),%0A%20%20created_at%20Instant,%0A%20%20updated_at%20Instant,%0A%20%20flagUrl%20String,%0A%20%20active%20Boolean,%0A%20%20nameEn%20String,%0A%20%20nameFr%20String,%0A%20%20nameEs%20String,%0A%20%20namePt%20String,%0A%20%20nameRu%20String%0A%7D%0Aentity%20DoubleMatch%20(doubles)%20%7B%0A%20%20event%20String%20required,%0A%20%20name%20String,%0A%20%20score%20String,%0A%20%20dateTime%20String%0A%7D%0Aentity%20Medal%20(medals)%20%7B%0A%20%20event%20String%20required,%0A%20%20name%20String,%0A%20%20winnerClass%20Integer,%0A%20%20loserClass%20Integer,%0A%20%20score%20String,%0A%20%20dateTime%20String%0A%7D%0Aentity%20MedalSummary%20(medal_summaries)%20%7B%0A%20%20rank%20Integer,%0A%20%20gold%20Float,%0A%20%20silver%20Float,%0A%20%20bronze%20Float%0A%7D%0Aentity%20SingleMatch%20(singles)%20%7B%0A%20%20event%20String%20required,%0A%20%20name%20String,%0A%20%20score%20String,%0A%20%20dateTime%20String%0A%7D%0Aentity%20ClassificationCard%20(classification_cards)%20%7B%0A%20%20classification%20Integer,%0A%20%20eligible%20Boolean,%0A%20%20created_at%20Instant,%0A%20%20updated_at%20Instant%0A%7D%0Aentity%20Player%20(players)%20%7B%0A%20%20nationalAssociation%20String,%0A%20%20country%20String%20required%20minlength(3)%20maxlength(3),%0A%20%20gender%20Gender,%0A%20%20classification%20Integer,%0A%20%20name%20String,%0A%20%20firstName%20String,%0A%20%20lastName%20String,%0A%20%20birthDate%20String,%0A%20%20eligible%20Boolean,%0A%20%20deceased%20Boolean,%0A%20%20yearFirstClassification%20Integer,%0A%20%20ittfId%20Integer,%0A%20%20otherName%20String,%0A%20%20remarks%20String%0A%7D%0Aentity%20PlayerProfile%20(profiles)%20%7B%0A%20%20name%20String,%0A%20%20gender%20Gender,%0A%20%20active%20Boolean,%0A%20%20otherName%20String,%0A%20%20dateOfBirth%20LocalDate,%0A%20%20handedness%20Handedness,%0A%20%20created_at%20Instant,%0A%20%20updated_at%20Instant,%0A%20%20deceased%20Boolean,%0A%20%20remarks%20String,%0A%20%20height%20Integer,%0A%20%20weight%20Integer,%0A%20%20birthCity%20String,%0A%20%20birthCountry%20String,%0A%20%20firstName%20String,%0A%20%20lastName%20String%0A%7D%0Aentity%20RatingMovement%20(rating_movements)%20%7B%0A%20%20seq%20Integer,%0A%20%20eventId%20String,%0A%20%20category%20String,%0A%20%20round%20String,%0A%20%20weight%20Float,%0A%20%20outcome%20Outcome%20required,%0A%20%20opponentRating%20Integer,%0A%20%20beforeRating%20Integer,%0A%20%20afterRating%20Integer%0A%7D%0Aentity%20TournamentPlayer%20(tournament_players)%20%7B%0A%20%20country%20String,%0A%20%20gender%20String,%0A%20%20classification%20Integer,%0A%20%20name%20String%0A%7D%0Aentity%20Rating%20(ratings)%20%7B%0A%20%20rating%20Integer,%0A%20%20classification%20Integer,%0A%20%20age%20String,%0A%20%20groupRank%20Integer,%0A%20%20classRank%20Integer,%0A%20%20juniorGroupRank%20Integer,%0A%20%20juniorClassRank%20Integer,%0A%20%20system%20Integer%0A%7D%0A%0Aenum%20Gender%20%7B%0A%20%20F,%0A%20%20M%0A%7D%0A%0Aenum%20Handedness%20%7B%0A%20%20LEFT,%0A%20%20RIGHT%0A%7D%0A%0Aenum%20Outcome%20%7B%0A%20%20L,%0A%20%20W%0A%7D%0A%0Arelationship%20ManyToOne%20%7B%0A%20%20Tournament%7BtournamentType(name)%7D%20to%20TournamentType,%0A%20%20Tournament%7BrankingList%7D%20to%20RankingList,%0A%20%20Country%7Bcontinent(name_en)%20required%7D%20to%20Continent,%0A%20%20DoubleMatch%7Bwinner1%20required%7D%20to%20ClassificationCard,%0A%20%20DoubleMatch%7Bwinner2%20required%7D%20to%20ClassificationCard,%0A%20%20DoubleMatch%7Bloser1%20required%7D%20to%20ClassificationCard,%0A%20%20DoubleMatch%7Bloser2%20required%7D%20to%20ClassificationCard,%0A%20%20DoubleMatch%7Btournament%20required%7D%20to%20Tournament,%0A%20%20Medal%7Bplayer(name)%20required%7D%20to%20Player,%0A%20%20Medal%7Btournament%20required%7D%20to%20Tournament,%0A%20%20MedalSummary%7Btournament(name)%20required%7D%20to%20Tournament,%0A%20%20MedalSummary%7Bcountry(code)%20required%7D%20to%20Country,%0A%20%20SingleMatch%7Bwinner%20required%7D%20to%20ClassificationCard,%0A%20%20SingleMatch%7Bloser%20required%7D%20to%20ClassificationCard,%0A%20%20SingleMatch%7Btournament%20required%7D%20to%20Tournament,%0A%20%20ClassificationCard%7BplayerProfile(name)%20required%7D%20to%20PlayerProfile,%0A%20%20PlayerProfile%7Bcountry(code)%20required%7D%20to%20Country,%0A%20%20RatingMovement%7BrankingList%7D%20to%20RankingList,%0A%20%20RatingMovement%7Bplayer(name)%20required%7D%20to%20Player,%0A%20%20RatingMovement%7Bopponent(name)%20required%7D%20to%20Player,%0A%20%20RatingMovement%7Btournament(name)%20required%7D%20to%20Tournament,%0A%20%20RatingMovement%7Bmatch%7D%20to%20SingleMatch,%0A%20%20TournamentPlayer%7Bplayer(name)%7D%20to%20Player,%0A%20%20TournamentPlayer%7Btournament(name)%7D%20to%20Tournament,%0A%20%20Rating%7BrankingList%7D%20to%20RankingList,%0A%20%20Rating%7Bplayer(name)%7D%20to%20Player%0A%7D%0A%0Adto%20RankingList%20with%20mapstruct%0Apaginate%20RankingList%20with%20pagination%0Aservice%20RankingList%20with%20serviceClass%0Adto%20TournamentType%20with%20mapstruct%0Aservice%20TournamentType%20with%20serviceClass%0Adto%20Tournament%20with%20mapstruct%0Apaginate%20Tournament%20with%20pagination%0Aservice%20Tournament%20with%20serviceClass%0Adto%20Continent%20with%20mapstruct%0Apaginate%20Continent%20with%20pagination%0Aservice%20Continent%20with%20serviceClass%0Adto%20Country%20with%20mapstruct%0Apaginate%20Country%20with%20pagination%0Aservice%20Country%20with%20serviceClass%0Adto%20DoubleMatch%20with%20mapstruct%0Apaginate%20DoubleMatch%20with%20pagination%0Aservice%20DoubleMatch%20with%20serviceClass%0Adto%20Medal%20with%20mapstruct%0Apaginate%20Medal%20with%20pagination%0Aservice%20Medal%20with%20serviceClass%0Adto%20MedalSummary%20with%20mapstruct%0Apaginate%20MedalSummary%20with%20pagination%0Aservice%20MedalSummary%20with%20serviceClass%0Adto%20SingleMatch%20with%20mapstruct%0Apaginate%20SingleMatch%20with%20pagination%0Aservice%20SingleMatch%20with%20serviceClass%0Adto%20ClassificationCard%20with%20mapstruct%0Apaginate%20ClassificationCard%20with%20pagination%0Aservice%20ClassificationCard%20with%20serviceClass%0Adto%20Player%20with%20mapstruct%0Apaginate%20Player%20with%20pagination%0Aservice%20Player%20with%20serviceClass%0Adto%20PlayerProfile%20with%20mapstruct%0Apaginate%20PlayerProfile%20with%20pagination%0Aservice%20PlayerProfile%20with%20serviceClass%0Adto%20RatingMovement%20with%20mapstruct%0Apaginate%20RatingMovement%20with%20pagination%0Aservice%20RatingMovement%20with%20serviceClass%0Adto%20TournamentPlayer%20with%20mapstruct%0Apaginate%20TournamentPlayer%20with%20pagination%0Aservice%20TournamentPlayer%20with%20serviceClass%0Adto%20Rating%20with%20mapstruct%0Apaginate%20Rating%20with%20pagination%0Aservice%20Rating%20with%20serviceClass%0A)